pyproject.toml

```diff
@@ -39,7 +39,7 @@ include = ['dodiscover*']
 exclude = ['*tests', '*testdata']
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
+python = ">=3.8.1,<3.11"
```

Comment:
Is this change necessary?

Reply:
This is requried by pygam 0.9.0

Reply:
Feel free to resolve convos if they are addressed.